exaudfclient/base/javacontainer/test/cpp/javavm_test.cc

```diff
@@ -32,6 +32,6 @@ void JavaVMTest::run(std::string scriptCode, SwigFactoryTestImpl & swigFactory)
     javaVMInternalStatus.m_classpath = javaVMImpl.m_classpath;
     javaVMInternalStatus.m_jvmOptions = javaVMImpl.m_jvmOptions;
     javaVMInternalStatus.m_needsCompilation = javaVMImpl.m_needsCompilation;
-    delete script_code;
+    ::free(script_code);
 }
```

Comment:
Huh?

Reply:
Reported by asan :)
`script_code` was allocated with `::malloc` (strdup) but deallocated with `delete`.
If you allocate with `::malloc` you should deallocated with `::free`.
If you allocate with `new` you should delete with `delete`